### ISSUE TITLE
Optional Parameter to use already Existing Collections instead of Creating them 

### ISSUE THREAD

**Issue by Thundereye90** [Maintainerr/Maintainerr#338]
Hi, 

Is it possible to add to the rules already existing collections?

For example I created a Collection using Plex Meta Manager which holds the Titles which I want to remove . 
The thing I created this with PMM is that PMM allows me to do fine grained conditions on which media I want to include in this Collection. 

So having another Program which creates Collections seems like a overkill. 

TLDR: Best practice would be an optional Field for Already Existing Collections to use instead of creating one.

**Comment by jorenn92** [comment]
Great suggestion, i'll look into it!

**Comment by jorenn92** [comment]
Possibilty added to specify a manual collection & to disable the rule system when creating rules.
Maintainerr will then sync with the Plex collection.

Until i perform a release, it'll only be available on the 'jorenn92/maintainerr:develop' docker tag.
